new-site/src/docs/elements/components/file-input/usage.mdx

```diff
@@ -8,36 +8,49 @@ import { FileInput } from 'hds-react';
 ## Usage
 
 ### Example
-<FileInput
-  multiple
-  dragAndDrop
-  id="file-input"
-  label="Drag and drop files here"
-  language="en"
-  accept=".png,.jpg"
-/>
+
+export const FileInputExample = () => {
```

Comment:
Do we need the export?

Reply:
Yes, if we want to use the same block in the markdown.
It seems to be the only way to use stateful components inline in markdown.